aggregates/governance-aggr/src/main/java/com/bloxbean/cardano/yaci/store/governanceaggr/processor/DRepVoteProcessor.java

```diff
@@ -0,0 +1,12 @@
+package com.bloxbean.cardano.yaci.store.governanceaggr.processor;
+
+import lombok.RequiredArgsConstructor;
```

Comment:
Thanks @Sotatek-HuyLe3a 

Do we need governanceaggr module in this PR?

Reply:
I have pushed this class unnecessarily.